app.rb

```diff
@@ -0,0 +1,274 @@
+require 'pry'
+# rubocop:disable Style/For
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/ModuleLength, Metrics/MethodLength
+module Enumerable
+  def my_each
```

Comment:
## my_each:

please make sure that your function return value matched the original one.

```ruby
  range = Range.new(5, 50)
  block = proc { |num| num < 9 / 2 }
  
  range.my_each(&block) === range.each(&block) => false

```